pepper-apis/docs/specification/src/components/responses.yml

```
@@ -32,6 +33,23 @@ ActivityAnswersPatchResponse:
                 answerGuid:
                   type: string
                   description: identifier for the answer
+          equations:
+            type: array
+            description: list of computed equations
```

Minor tweak to the wording with the intent to make it more explicit about what's being included here.

```suggestion
            description: list of computed equation values
```